script.js

```diff
@@ -1,34 +1,39 @@
+//Calculate Tip
+
 function calculateTip() {
-  var billTot = document.getElementById("billTot").value;
-  var tipPer = document.getElementById("tipPer").value;
-  var split = document.getElementById("split").value;
-
- //validate input
- if (billTot === "" || serviceQual == 0) {
-  alert("Please enter values");
-  return;
-}
+    var billAmt = document.getElementById("billAmt").value;
+    var servicequality = document.getElementById("servicequality").value;
+    var billsplit = document.getElementById("billsplit").value;
```

Comment:
Use `camelUse` when naming variables that have multiple words in them. Or if you really don't want to do that make sure all your variable names are consistently not doing that. `competeTotal` is camelCase but the rest aren't. Consistency is important.